Source/non_ip/domestic.ts

```diff
@@ -341,6 +341,41 @@ export const DOMESTICS: Record<string, DNSMapping> = {
       'baomitu.com',
       'qiku.com'
     ]
+  },
+  NETEASE: {
+    hosts: {},
+    dns: 'https://doh.pub/dns-query',
+    realip: false,
+    ruleset: true,
+    domains: [
+      '126.com',
+      '126.net',
+      '127.com',
+      '127.net',
+      '+163.com',
+      '163yun.com',
+      '+163.com.cn',
+      '163jiasu.com',
+      '166.com',
+      '166.net',
+      '188.com',
+      'netease.com',
+      '+neteasecloudmusic.com',
+      '+neteasegame.com',
+      '+netease.im',
+      'netstatic.net',
+      'lofter.com',
+      'yeah.net',
+      'ydstatic.com',
+      'youdao.com',
+      'youdao.cn',
+      'yoozoo.com',
+      'yqkk.link',
+      'haowu.link',
+      'laiqukankan.com',
+      'zhuanzfx.com',
+      '321fenx.com'
+    ]
```

Comment:
Why DNSPod? NetEase uses multiple CDNs from different providers. For instance, NetEase Music primarily uses Alibaba Cloud CDN.

Reply:
> Why DNSPod? NetEase uses multiple CDNs from different providers. For instance, NetEase Music primarily uses Alibaba Cloud CDN.

I have never expected that. I didn't found public records saying NetEase have used multiple CDNs for different services.


>https://github.com/mawenjian/china-cdn-domain-whitelist/blob/master/china-top-website-whitelist.conf

I wrote the update from this record.

Reply:
> I have never expected that. I didn't found public records saying NetEase have used multiple CDNs for different services.

`m*.music.126.net` uses both Alibaba Cloud CDN and China Telecom's CDN.
`p*.music.126.net` uses both Bytedance (volcengine) CDN and China Telecom's CDN.